java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/logic/impl/UsermanagementDummyImpl.java

```diff
@@ -17,16 +19,41 @@
 @Component
 public class UsermanagementDummyImpl extends AbstractBeanMapperSupport implements Usermanagement {
 
+  @Inject
+  private com.devonfw.application.mtsj.usermanagement.logic.api.Usermanagement usermanagementcore;
+
   @Override
   public UserProfile findUserProfileByLogin(String login) {
 
+    UserEto userEto = this.usermanagementcore.findUserbyName(login);
     // this is only a dummy - please replace with a real implementation
     UserDetailsClientTo profile = new UserDetailsClientTo();
+    profile.setId(userEto.getId());
     profile.setName(login);
-    profile.setFirstName("Peter");
-    profile.setLastName(login);
-    profile.setRole(Role.WAITER);
+    /*
+     * profile.setFirstName("Peter"); profile.setLastName(login);
+     */
+    profile.setRole(setRole(userEto.getUserRoleId()));
     return profile;
   }
 
+  /**
+   * Returns a Role instance from its id
+   *
+   * @param roleId received from the {@link UserEto constructed in findUserProfileByLogin}
+   * @return an instance from {@link Role}
+   */
+  private Role setRole(long roleId) {
```

Comment:
What do we need this for?
How about using `Enum.ordinal()` ?
Why do we need numeric representation of the Role at all?

Reply:
Actually this is a Dummy class, so now the hard coded Role.waiter is removed and it set the role according to login